openedx-unsupported/frontend-app-support-tools
# feat:add SSO history to the SSO card

This PR introduces a history section in SSO Cards incoming from backend with the SSO support API endpoint. 
Ticket Link: [PROD-2423](https://openedx.atlassian.net/browse/PROD-2423)

Testing method: 
- Go to [LMS admin](http://localhost:18000/admin/social_django/usersocialauth/) and add an SSO record or update a SSO Record. 
- Go to SSO tab in [learner information](http://localhost:18450/learner_information/) inside support tools and search for the relevant username. Click on 'See History' button on the card. 

Link to the [backend PR](https://github.com/openedx/edx-platform/pull/30418)

Screenshot:
<img width="1483" alt="Screenshot 2022-06-06 at 5 18 35 PM" src="https://user-images.githubusercontent.com/56605828/172159284-f482dc6e-0ad7-4854-b8ad-c0fe7a917533.png">
